File: unique.js
function unique(string){
    const array = string.split("");

    for(let i = 0; i < array.length - 1; i++){
        for(let j = i + 1; j < array.length; j++){
            if(array[i] === array[j])
                return false;
        }
    }

    return true;
}

console.log(unique("abcdefghijklmnopqrstuvwxyz"));
console.log(unique("abcdefghijklmnopqrstuvwxyzz"));
console.log(unique("aa"));
console.log(unique("a"));
console.log(unique(""));

File: go/unique.go
package main

import "fmt"

func unique(input string) bool {
	array := []rune(input)

	for i := 0; i < len(array)-1; i++ {
		for j := i + 1; j < len(array); j++ {
			if array[i] == array[j] {
				return false
			}
		}
	}

	return true
}

func main() {
	fmt.Println(unique("abcdefghijklmnopqrstuvwxyz"))
	fmt.Println(unique("abcdefghijklmnopqrstuvwxyzz"))
	fmt.Println(unique("aa"))
	fmt.Println(unique("a"))
	fmt.Println(unique(""))
}
